Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN cd $REPO_PATH && \
     make get_vendor_deps && \
     make test_unit && \
     make build_linux && \
+    source scripts/setTestEnv.sh && \
     make install && \
```

Comment:
Somebody may prefer run mainnet with docker. How about prepare two docker file: one for testnet and another one for mainnet?